### ISSUE TITLE
[3.0.0-rc1] Deprecated warnings in setup

### ISSUE THREAD

**Issue by alexsoin** [modxcms/revolution#15988]
### Summary

During installation, after passing the database check, an error is shown:

_(link of the page where the error is shown https://url.com/setup/?action=database)_

```
Deprecated: Return type of xPDO\xPDOMap::offsetExists($offset) should either be compatible with ArrayAccess::offsetExists(mixed $offset): bool, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /home/public_html/core/vendor/xpdo/xpdo/src/xPDO/xPDOMap.php on line 33 
Deprecated: Function strftime() is deprecated in /home/public_html/setup/includes/modinstall.class.php on line 479
Deprecated: Function strftime() is deprecated in /home/public_html/setup/includes/modinstall.class.php on line 203
Warning: Cannot modify header information - headers already sent by (output started at /home/public_html/core/vendor/xpdo/xpdo/src/xPDO/xPDOMap.php:16) in /home/public_html/setup/includes/request/modinstallrequest.class.php on line 228 
```

### Environment
MODX version 3.0.0-rc1
I tried to install both on PHP 7.4 and on 8.1

Download the installer from [here](https://modx.com/download/other-versions?releaseKey=revolution-3.0.0-rc1).

**Comment by JoshuaLuckers** [comment]
@opengeek any idea why the deprecated warning is still being shown? Or didn't rc1 get the changes from https://github.com/modxcms/xpdo/pull/219 ?

**Comment by JoshuaLuckers** [comment]
`Deprecated: Function strftime() is deprecated` is related to issue #15864

**Comment by opengeek** [comment]
> @opengeek any idea why the deprecated warning is still being shown? Or didn't rc1 get the changes from [modxcms/xpdo#219](https://github.com/modxcms/xpdo/pull/219) ?

It definitely should have. I'm confused.

**Comment by opengeek** [comment]
It looks like I missed the offsetExists() warning. Sigh.

**Comment by opengeek** [comment]
The xPDOMap issue is resolved via modxcms/xpdo#220.

**Comment by Ibochkarev** [comment]
Fixed #15999
